tests/test_zenhub.py

```diff
@@ -46,28 +46,29 @@ def json(self):
         assert False
 
 
-
 class TestZenHub(unittest.TestCase):
 
     @patch('src.zenhub.ZenHub._generate_url')
     @patch('src.zenhub.ZenHub._get_repo_id')
-    @patch('src.zenhub.ZenHub._get_token')
     @patch('requests.get', side_effect=mocked_response)
-    def test_happy_path(self, mocked_get_info, mock_get_token, mock_repo_id, mock_generate_url):
-        path_to_token = '~/foo/bar/baz.txt'
+    def test_happy_path(self, mocked_get_info, mock_repo_id, mock_generate_url):
         repo_name = 'azul'
         issue = 42
 
-        mock_get_token.return_value = 99999999
         mock_repo_id.return_value = 123456789
         mock_generate_url.return_value = (
             f"https://api.zenhub.io/p1/repositories/{mock_repo_id.return_value}/issues/{issue}")
 
-        res = ZenHub(path_to_token=path_to_token,
-                     repo_name=repo_name,
-                     issue=issue)
+        class TestZenHub(ZenHub):
```

Comment:
This is kind of confusing to have a TestZenHub class within another TestZenHub class.

Reply:
I agree with Ben here

Reply:
The purpose of the class inside the test class is to change one of the parameters set in the constructor `def __init__`. Tried a few things, but was difficult (wasn't able to patch it). I changed the name of the inner class (which inherits from the actual class we want to test). I agree that the original name of the inner class was ill-chosen. Otherwise I would say we leave it for now as it does the trick.

Reply:
Sounds good but should be done for all of the other class definitions I pointed out too.

Reply:
Also, see the comment I made in the diff. I should have caught that earlier.

Reply:
done